api/generated/@tanstack/react-query.gen.ts

```diff
@@ -178,7 +178,6 @@ export const getApiOpenapiJsonQueryKey = (
 
 /**
  * Get OpenAPI specification
- *
```

Comment:
I think you need to do a `pnpm i` first

Reply:
you are right :+1: 

I was wondering why it happened...